scripts/issues.coffee

```diff
@@ -66,3 +73,27 @@ module.exports = (robot) ->
           msg.send "\##{issue.number} @#{repositoryFullName} - #{issue.title} - #{issue.state} - #{issue.html_url}"
         else
           robot.logger.error "Repo #{repositoryFullName} not found, or undefined"
+
+  robot.hear /https:\/\/github\.com\/(\w+)\/(\w+)\/issues\/(\d+)/gi, (msg) ->
```

Comment:
We want this to work with PRs too, e.g. `https://github.com/MovingBlocks/GooeyJr/pull/56/`
There's no need to change the API call because GitHub treats both issues and PRs the same when retrieving information. You just need to change the regex pattern so it the bot gets triggered for PR urls too. I believe the change required is `\/issues\/` -> `\/(?:issues | pull)\/`.

Let's also drop the `https:\/\/` at the start of the regex pattern so it matches URLs more broadly.